AI/scripts/util/visualization_utils.py

```diff
@@ -0,0 +1,97 @@
+"""
+Provides utilities for visualization.
+Includes visualization of bounding boxes for annotations/detections.
+"""
+
+__author__ = 'Ian Randman'
+
+import logging
+
+import numpy as np
+import pandas as pd
+from object_detection.utils import visualization_utils as viz_utils
+
+import matplotlib.pyplot as plt
+import matplotlib as mpl
+
+# may fail depending on installation of matplotlib
+mpl.use('module://backend_interagg')
```

Comment:
What if it does fail? How would you fix it?

Reply:
Change it

Reply:
Like if I were working on this project in the future and ran into this I would be annoyed with that comment but as long as it's not an issue now I guess it's not like the end of the world. Is there anything we can say to clarify in what way it could be changed to fix it?

Reply:
I changed it to make it less likely to fail, I think. But matplotlib is a weird package and the backend to use is kinda depending on your system. Please see my change.